src/ufoLib2/objects/layerSet.py

```diff
@@ -58,15 +70,15 @@ class LayerSet:
             del font.layers["myLayerName"]
     """
 
-    _layers: MutableMapping[str, Layer] = attr.ib(
+    _layers: MutableMapping[str, Union[Layer, Placeholder]] = attr.ib(
         factory=OrderedDict, converter=_convert_layers
     )
     defaultLayer: Optional[Layer] = None
```

Comment:
ah.. we initialize defaultLayer in `__attrs_post_init__` that's why mypy complains about it being Optional all the time.. But it's never optional really, only for a brief moment.
I wish wish we didn't have to do all this checking for None everywhere we use defaultLayer